### ISSUE TITLE
Warnings with tests

### ISSUE THREAD

**Issue by srmnitc** [ross-rotordynamics/ross#472]
This is part of my review of the software paper at openjournals/joss-reviews#2120

The tests run as described,  the tests raise a fair number of `DeprecationWarning` (18 ) and skips 6 tests (from `test_rotor_assembly.py` and `test_units.py`).  The first set in from `results.py` - a lot of warnings for invalid escape sequence and the rest are from Bokeh. I will it would be great and potentially would remove a lot of confusion is these are prevented.

**Comment by rodrigomoliveira1** [comment]
Hello @srmnitc 

Those `DeprecationWarning`'s about invalid escape sequence happens  while using LaTeX syntax to write greek letters in matplotlib plot. And about Bokeh, they're happening due their recent updates to version 1.4.0 and 2.0.0. I'm going to fix most of them.

About the tests:
- ``test_rotor_assembly.py`: we've skipped some tests that returns matrices with very small values (e.g. 1e-18, 1e-19). As the computational precision does not exceed 16 decimal digits, each time the tests are run, these values appear different (although this does not imply wrong final results). In fact, we have added some explaination on the command line why we're skipping these tests.

- `test_units.py`: the tests we're skipping are related to future features. So as soon we implement them, we'll turn these tests on back again.
